.github/workflows/pr.yml

```diff
@@ -63,6 +63,36 @@ jobs:
 
       - run: make test ci
 
+      - name: Build without default features
+        run: cargo check --no-default-features
+
+      - name: Build with only duckdb
+        run: cargo check --no-default-features --features duckdb
```

Comment:
can this be in a matrix?

Reply:
Putting it into a matrix will run a separate job per matrix item, whereas in theory this can re-use the existing build cache to go faster.